src/Microsoft.AspNetCore.Authentication.Facebook/FacebookOptions.cs

```diff
@@ -57,6 +60,6 @@ public string AppSecret
         /// The list of fields to retrieve from the UserInformationEndpoint.
         /// https://developers.facebook.com/docs/graph-api/reference/user
         /// </summary>
-        public IList<string> Fields { get; }
+        public ICollection<string> Fields { get; } = new HashSet<string>();
```

Comment:
No particular scenario in mind. But exposing `ISet` in the public contract makes it clear that the collection cannot have duplicate items (which is not something `IList`/ `ICollection` imply).